parm/use_cases/met_tool_wrapper/GenVxMask/GenVxMask_multiple.conf

```diff
@@ -0,0 +1,75 @@
+[config]
+## Configuration-related settings such as the process list, begin and end times, etc.
+
+# List of applications to run - only GenVxMask for this case
+PROCESS_LIST = GenVxMask
+
+# time looping - options are INIT, VALID, RETRO, and REALTIME
+# If set to INIT or RETRO:
+#   INIT_TIME_FMT, INIT_BEG, INIT_END, and INIT_INCREMENT must also be set
+# If set to VALID or REALTIME:
+#   VALID_TIME_FMT, VALID_BEG, VALID_END, and VALID_INCREMENT must also be set
+LOOP_BY = INIT
+
+# Format of VALID_BEG and VALID_END using % items
+# %Y = 4 digit year, %m = 2 digit month, %d = 2 digit day, etc.
+# see www.strftime.org for more information
+# %Y%m%d%H expands to YYYYMMDDHH
+INIT_TIME_FMT = %Y%m%d%H
+
+# Start time for METplus run - must match INIT_TIME_FMT
+INIT_BEG = 2005080700
+
+# End time for METplus run - must match INIT_TIME_FMT
+INIT_END = 2005080700
+
+# Increment between METplus runs (in seconds if no units are specified)
+#  Must be >= 60 seconds
+INIT_INCREMENT = 1M
+
+# List of forecast leads to process for each run time (init or valid)
+# If unset, defaults to 0 (don't loop through forecast leads
+LEAD_SEQ = 24H
+
+# Order of loops to process data - Options are times, processes
+# Not relevant if only one item is in the PROCESS_LIST
+# times = run all wrappers in the PROCESS_LIST for a single run time, then
+#   increment the run time and run all wrappers again until all times have
+#   been evaluated.
+# processes = run the first wrapper in the PROCESS_LIST for all times
+#   specified, then repeat for the next item in the PROCESS_LIST until all
+#   wrappers have been run
+LOOP_ORDER = times
+
+# Verbosity of MET output - overrides LOG_VERBOSITY for GenVxMask only
+LOG_GEN_VX_MASK_VERBOSITY = 2
+
+GEN_VX_MASK_SKIP_IF_OUTPUT_EXISTS = False
+
+# Time relative to valid time (in seconds if no units are specified) to allow files to be considered
+#  valid. Set both BEGIN and END to 0 to require the exact time in the filename
+#  Not used in this example.
+GEN_VX_MASK_FILE_WINDOW_BEGIN = 0
+GEN_VX_MASK_FILE_WINDOW_END = 0
+
+GEN_VX_MASK_OPTIONS = -type lat -thresh 'ge30&&le50', -type lon -thresh 'le-70&&ge-130' -intersection
+
+[filename_templates]
+
+# Template to look for input to GenVxMask relative to GEN_VX_MASK_INPUT_DIR
+GEN_VX_MASK_INPUT_TEMPLATE = {INPUT_BASE}/met_test/data/sample_fcst/{init?fmt=%Y%m%d%H}/wrfprs_ruc13_{lead?fmt=%2H}.tm00_G212
+
+#GEN_VX_MASK_INPUT_MASK_TEMPLATE = {GEN_VX_MASK_INPUT_TEMPLATE}, {GEN_VX_MASK_INPUT_TEMPLATE}
```

Comment:
Unclear why this commented out line remains.  Can it be deleted?  If not, add a comment explaining why its there.

Reply:
This was done in a test to see if an arbitrary string would work in place of an actual filename. The test worked, so I will remove the commented line.